amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/resources-alignment.yaml

```diff
@@ -0,0 +1,4 @@
+ramMin: 60000
+coresMin: 15
+tmpdirMin: 225000
+outdirMin: 225000
```

Comment:
This and the following files prefixed `resources-` get pulled into their respective workflows using the `$mixin` directive. The point here is that we will be able to create different resource files for different collections of samples / sizes of samples.